app/assets/stylesheets/tmdb.css

```diff
@@ -0,0 +1,137 @@
+.cast-container {
```

Comment:
Can we call this something other than TMDB? That doesn't really tell me what is in here.

Reply:
renamed to `cast.css` as discussed